### PR TITLE
adding stateset to data-hub

### PR DESCRIPTION
**Pull request by domsteil**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1418,5 +1418,41 @@
...
+		  {
+			"url": "",
+			"alt": ""
+		  }
+		],
+		"video_demo": "",
+		"integration_link": "github.com/stateset/data-hub/stateset/README.md",
+		"categories": [
+		  {
+			"name": "eCommerce"
```
Comment (praveenweb) [inline config.json:1451]: Can we change this to Ecommerce to keep it consistent with other similar integrations.

File: config.json [hunk 2]
```diff
@@ -1418,5 +1418,41 @@
...
+		  }
+		],
+		"is_featured": false,
+		"coming_soon": false,
+		"type": "Database Schema"
```
Comment (praveenweb) [inline config.json:1456]: The type should be `Remote Schema` from what I understand of this integration :)

**Comment by praveenweb** [comment]
@domsteil - Hey! Thanks for adding Stateset. I have left a couple of comments that can be quickly addressed before we merge this :)

**Comment by domsteil** [comment]
@praveenweb Just made the updates, thanks!

**Pull request merged**